packages/manager/apps/telecom/src/components/notification/notification-list.html

```diff
@@ -29,6 +56,14 @@ <h2 data-translate="components_notification_loading"></h2>
                         {{:: 'components_notification_type' | translate }}
                     </tuc-col-sort>
                 </th>
+                <th scope="col">
```

Comment:
Out of scope of this PR but we should think to turn this into a oui-datagrid

Reply:
Good idea. I note this for a future task.